DESCRIPTION

```diff
@@ -1,14 +1,22 @@
 Package: geohash
-Title: What the Package Does (One Line, Title Case)
+Title: Functions for geohash encoding and decoding
+Description: A way to use a set of wrappers around 'rgeohash' rust crate. It includes encoding and decoding functions and a recipe step_geocode for geohash encoding.
 Version: 0.0.0.9000
-Authors@R: 
-    person("Josiah", "Parry", , "josiah.parry@gmail.com", role = c("aut", "cre"),
-           comment = c(ORCID = "0000-0001-9910-865X"))
-Description: What the package does (one paragraph).
+Authors@R: c(
+        person("Josiah", "Parry", email= "josiah.parry@gmail.com",
+            role = c("aut", "cre"), comment = c(ORCID = "0000-0001-9910-865X")
+        ))
 License: MIT + file LICENSE
 Encoding: UTF-8
 Language: en
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.3.1
+RoxygenNote: 7.3.2
 Config/rextendr/version: 0.3.1.9001
 SystemRequirements: Cargo (Rust's package manager), rustc >= 1.70
+Imports: 
+    cli,
+    recipes,
+    rlang (>= 1.1.0)
```

Comment:
It is really important to me that we keep the package dependency free. These should all be moved to suggests